### PR TITLE
Design branch

### PR DESCRIPTION
**Pull request by KmBarnett**
# Description :: User Story

## Type of change

- [x] New Feature
- [x] Bug Fix
- [ ] Refactor
- [ ] Breaking Change

## Notes
This PR adds style to the page and addresses some issues with var names.
## Test results
passing

### CODE REVIEW

File: src/Components/ChatContainer/ChatContainer.js [hunk 1]
```diff
@@ -12,34 +12,27 @@ const endPoint = "http://localhost:5000";
...
   useEffect(() => {
     socket.on("join_room",function(data) {


-      setRoomData(data)
+      setRoomDetails(data)
       socket.emit('recived', true)
     });
-  },[roomData])
+  },[roomDetails])
```
Comment (Yetidancer) [inline src/Components/ChatContainer/ChatContainer.js]: getting a little dejá vu here

File: src/Components/ChatContainer/ChatContainer.test.js [hunk 2]
```diff
@@ -80,6 +85,29 @@ describe("<ChatContainer />", () => {
...
+
+
+    socket.on('message', function (message) {
+      socket.emit('message', message)
+     });
+
+    fireEvent.change(messageInput, { target: { value: 'Test' } })
+    fireEvent.click(messageSend)
+
+    socket.on('recived', function (recived) {
```
Comment (Yetidancer) [inline src/Components/ChatContainer/ChatContainer.test.js]: sp

File: src/Components/App/App.js [hunk 3]
```diff
@@ -21,13 +23,22 @@ function App({ grabLocalCodes }) {
...
-  useEffect(() => {
+  useLayoutEffect(() => {
     storeMyCodes()
-  }, [])
+    let landing = window.location.pathname === '/' ?
+    'main-page' : 'not-main'
+    setLocation(landing)
+  },[])
+
+  history.listen((location) => {
```
Comment (Yetidancer) [inline src/Components/App/App.js:37]: what's history.listen do?

File: src/Components/Participant/Participant.js [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
+//
   const displayParticipant = ( message ) => {
     return (
       <section>
-        <h3>Circle Code: { group }</h3>
+        <section className='code-section'>
+          <h3>Circle Code: { group }</h3>
+          <CopyToClipboard text={`${window.location.origin}/join/${group}`}>
+            <button id='icon-button' data-tooltip="Copy Invite Link.">
+              <img className='icon' src="assets/clipboard.svg" alt="clipboard" />
```
Comment (Yetidancer) [inline src/Components/Participant/Participant.js]: good work on the clipboard stuff!

File: src/Components/ChatContainer/ChatContainer.js (hunk 1 above)
Comment (KmBarnett) [reply src/Components/ChatContainer/ChatContainer.js]: yeah think we both changed this

File: src/Components/ChatContainer/ChatContainer.test.js (hunk 2 above)
Comment (KmBarnett) [reply src/Components/ChatContainer/ChatContainer.test.js]: sp?

File: src/Components/App/App.js (hunk 3 above)
Comment (KmBarnett) [reply src/Components/App/App.js:37]: it is listening to the URL and running a function when it changes

**Pull request closed without merging**